gettsim/transfers/rente.py

```diff
@@ -251,21 +259,31 @@ def ges_rente_zugangsfaktor(
         alter_renteneintritt = jahr_renteneintr - geburtsjahr
 
         # Calc difference to Regelaltersgrenze
-        diff = alter_renteneintritt - ges_rente_regelaltersgrenze
+        # (todo: replace ges_rente_regelaltersgrenze
+        # with ar_langjährig_versicherte)
+        diff_a = alter_renteneintritt - ges_rente_grenze_altersrente
+        diff_l = (
```

Comment:
It is not directly intuitive what the meaning of the three different `diff_` variables is. I would use longer variable names and/or explain in a comment how they differ.

Reply:
changed the terms to `diff_regelrente` `diff_longterm_pension_age`, and `diff_volle_rente`.